bazel/repository_locations.bzl

```diff
@@ -433,10 +433,11 @@ DEPENDENCY_REPOSITORIES_SPEC = dict(
     com_google_googletest = dict(
         project_name = "Google Test",
         project_url = "https://github.com/google/googletest",
-        version = "1.10.0",
-        sha256 = "9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb",
-        strip_prefix = "googletest-release-{version}",
-        urls = ["https://github.com/google/googletest/archive/release-{version}.tar.gz"],
+        # 2020-09-10
```

Comment:
Thanks for the update. Please include an explanation as per policy, see https://github.com/envoyproxy/envoy/blob/master/DEPENDENCY_POLICY.md

> Versions should prefer release versions over master branch GitHub SHA tarballs. A comment is necessary if the latter is used. This comment should contain the reason that a non-release version is being used and the YYYY-MM-DD when the last update was performed.

Reply:
@wrowe ping can you add the comment?

/wait

Reply:
Indeed, I had misunderstood the exception r.e. CPE's and test-only elements of the build, but hope you find the new message appropriate. Not really certain if there will be a 1.11 "release" given the general google philosophy of consuming the dev branch, and with respect to our test architecture, any of their bleed efforts to identify defects should be picked up fairly frequently (although probably not daily.)